src/functions/vacation-request/update-vacation-request/handler.ts

```diff
@@ -29,11 +30,11 @@ const updateVacationRequestHandler: ValidatedEventAPIGatewayProxyEvent<typeof va
   if (!id) {
     return {
       statusCode: 400,
-      body: "Bad request, missing id"
+      body: "Bad vacation request, missing id"
     };
   }
 
-  if(!personId || !draft || !startDate || !endDate || !days || !type || !message || !createdBy || !createdAt || !updatedAt || !updatedBy) {
+  if(!personId || !draft || !startDate || !endDate || !days || !type || !status || !message || !createdBy || !createdAt || !updatedAt || !updatedBy) {
```

Comment:
Does this mean you are opting to have status as an attribute within vacation rather than a separate entity?

If so I think this can work but we need to discuss the specific requirements for updating the status before we go ahead with this